Framework/Algorithms/src/Stitch1D.cpp

```diff
@@ -3,6 +3,7 @@
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/WorkspaceProperty.h"
 #include "MantidAlgorithms/RunCombinationHelpers/RunCombinationHelper.h"
+#include "MantidAlgorithms/SortXAxis.h"
```

Comment:
I don't think this `#include` is needed anymore.

Reply:
It wasn't good spot!